native/src/rapidsmlJniRemain.cu

```diff
@@ -0,0 +1,149 @@
+#include <cuda_runtime.h>
```

Comment:
The `Remain` name in the file seems hacky and non-descript, and I'm a bit surprised to see the rapidsml_jni.cu file deleted.  Why not leave rapidsml_jni.cu in place and add rapmdsmlJni.cpp with the new dgemm stuff?  That would be a much smaller diff for this PR.

Reply:
Agree, renamed them back.